src/utils/rfqm_health_check.ts

```diff
@@ -0,0 +1,50 @@
+export enum HealthCheckStatus {
+    Operational = 'operational',
+    Unknown = 'unknown',
+    Maintenance = 'maintenance',
+    Degraded = 'degraded',
+    Failed = 'failed',
+}
+
+interface HealthCheckIssue {
+    status: HealthCheckStatus;
+    description: string;
+}
+
+export interface HealthCheckResult {
+    status: HealthCheckStatus;
+    pairs: {
```

Comment:
slight preference for array over key, value here to avoid needing to string parse the tokens out of the key

Reply:
Ack.

Thought here is that this enables pair-level checks/reporting... i.e. all the transactions on pair X have failed for the last hour. We mark it as `FAILED`, then just don't send it to the front end.